This moves the y-min/y-max parameters into ChannelSettingSnapshot

cnlklink opened the pull request.

cnlklink committed:
Moved the yMin and yMax into the ChannelSettingSnapshot

cnlklink committed:
Merge branch 'main' into move-ymin-ymax

rneswold reviewed (dismissed):
These changes look good but I need to add the GraphQL changes to this pull request before we merge.

rneswold committed:
:wrench: add the rest of the support

These changes tie-in John's changes with the GraphQL generator. We still
need to update the GraphQL service before this will work.

cnlklink commented:
After merging `main` into this branch it appears that we've broken `flutter-controls-plotting` -- https://github.com/fermi-ad/flutter-controls-plotting/actions/runs/17116328422/job/48548083605?pr=89

rneswold commented:
I guess we merge #92 and then merge this branch with `main` again.

cnlklink committed:
Merge branch 'main' into move-ymin-ymax

beauremus commented on lib/src/service/acsys_service.dart:
```diff
@@ -393,8 +393,15 @@ final class PlotPoint {
   final int? markerIndex;
-
-  const ChannelSettingSnapshot({this.lineColor, this.markerIndex});
+  final double? min;
```
Not a deal breaker, but saying yMin and yMax is most clear.

beauremus reviewed (dismissed):
LGTM! :shipit:

cnlklink committed:
Removed unused import

cnlklink committed:
REFACTOR - renamed yMin and yMax
